install.ps1

```diff
@@ -0,0 +1,142 @@
+    $pyVersion =  ([string]$pyVersion).substring(7,3)
+    if([double]$pyVersion -lt 3.5){
+        throw "python version should >= 3.5"
+    }
+}
+
+$WHICH_PIP = where.exe pip
+if($WHICH_PIP -eq $null){
+    throw "Can not find pip"
+}
+if($WHICH_PYTHON[0].Length -eq 1){
+    $NNI_PYTHON3 = $WHICH_PYTHON.SubString(0,$WHICH_PYTHON.Length-11)
+}
+else{
+    $NNI_PYTHON3 = $WHICH_PYTHON[0].SubString(0,$WHICH_PYTHON[0].Length-11)
+}
+$NNI_PKG_FOLDER = $NNI_PYTHON3 +"\nni"
+$PIP_INSTALL = "$NNI_PYTHON3\python -m pip install ."
+
+if(!(Test-Path $NNI_DEPENDENCY_FOLDER)){
+    New-Item $NNI_DEPENDENCY_FOLDER -ItemType Directory
+}
+$NNI_NODE_ZIP = $NNI_DEPENDENCY_FOLDER+"\nni-node.zip"
+$NNI_NODE_FOLDER = $NNI_DEPENDENCY_FOLDER+"\nni-node"
+$NNI_YARN_TARBALL = $NNI_DEPENDENCY_FOLDER+"\nni-yarn.tar.gz"
+$NNI_YARN_FOLDER = $NNI_DEPENDENCY_FOLDER+"\nni-yarn"
+$NNI_YARN = $NNI_YARN_FOLDER +"\bin\yarn"
+
+## Version number
+$NNI_VERSION_VALUE = $(git describe --tags)
+$NNI_VERSION_TEMPLATE = "999.0.0-developing"
+
+if(!(Test-Path $NNI_NODE_ZIP)){
+    Write-Host "Downloading Node..."
+    (New-Object Net.WebClient).DownloadFile($nodeUrl, $NNI_NODE_ZIP)
+}
+
+if(!(Test-Path $NNI_YARN_TARBALL)){
+    Write-Host "Downloading Yarn..."
+    (New-Object Net.WebClient).DownloadFile($yarnUrl, $NNI_YARN_TARBALL)
+}
+
+$NNI_YARN_TARBALL = $NNI_YARN_TARBALL -split '\\' -join '\\'
+$NNI_DEPENDENCY_FOLDER = $NNI_DEPENDENCY_FOLDER -split '\\' -join '\\'
+$SCRIPT_PATH = $NNI_DEPENDENCY_FOLDER + '\extract.py'
+$SCRIPT =  "import tarfile",
+       ("tar = tarfile.open(""{0}"")" -f $NNI_YARN_TARBALL),
+       ("tar.extractall(""{0}"")" -f $NNI_DEPENDENCY_FOLDER),
+        "tar.close()"
+[System.IO.File]::WriteAllLines($SCRIPT_PATH, $SCRIPT)
+
+if ($install_node) {
+    ### nodejs install
+    if(Test-Path $NNI_NODE_FOLDER){
+        Remove-Item $NNI_NODE_FOLDER -Recurse -Force
+    }
+	Expand-Archive $NNI_NODE_ZIP -DestinationPath $NNI_DEPENDENCY_FOLDER
+    $unzipNodeDir = Get-ChildItem "$NNI_DEPENDENCY_FOLDER\$unzipNodeDir"
+    Rename-Item $unzipNodeDir "nni-node"
+     
+    ### yarn install
+    if(Test-Path $NNI_YARN_FOLDER){
+        Remove-Item $NNI_YARN_FOLDER -Recurse -Force
+    }
+    cmd /C "$NNI_PYTHON3\python" $SCRIPT_PATH
+    $unzipYarnDir = Get-ChildItem "$NNI_DEPENDENCY_FOLDER\$unzipYarnDir"
+    Rename-Item $unzipYarnDir "nni-yarn"
+}
+
+### add to PATH
+function Add2Path {
+    param ($fileName)
+    $PathVariable = [System.Environment]::GetEnvironmentVariable("Path","User")
+    $PathFolders = $PathVariable.Split(";")
+    if(!$PathFolders.Contains($fileName)){
+        if($PathVariable.Trim().EndsWith(";")){
+            $PathVariable = $PathVariable + $fileName
+        }
+        else {
+            $PathVariable = $PathVariable + ";" + $fileName
+        }
+        [System.Environment]::SetEnvironmentVariable("Path",$PathVariable,"User")
+    }
+}
+
+Add2Path -fileName $NNI_NODE_FOLDER
+Add2Path -fileName "$NNI_YARN_FOLDER\bin"
```

Comment:
need to update path to find tsc, react-scripts-ts-antd

Reply:
Can refresh path in line 107 do it?